web/index.jsp

```diff
@@ -29,6 +29,10 @@ include file="projects.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
     cfg.setTitle("Search");
+    String sourceRootPath = cfg.getSourceRootPath();
```

Comment:
Yes this is an overkill. I'd patch it with the `canRead` method even though it's 100% valid on all systems. Also this question is for java6 and we have java8 now.